weblogic-azure-aks/pom.xml

```diff
@@ -8,14 +8,6 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <!-- mvn -Pbicep -Passembly clean install -Ptemplate-validation-tests -->
-    <!-- Build with development profile
-         If using azure-javaee-iaas-parent earlier than 1.0.13, use:
-           mvn -Pbicep -Passembly -Ddev clean install
-         Otherwise, use
-           mvn -Pbicep-dev -Passembly clean install
-    -->
-
```

Comment:
I understand your desire to remove this, but I and others rely on having a comment that explains how to run things. By removing this are you saying, "we no longer support building at any level other than at the root level, so don't try to build from here?"  If you are saying that, please add a comment that directs a reader to the top level so they can build it there.

Otherwise, please tell the reader what they need to do if they want to build just the AKS offer.